internal/reader/home_chain.go

```diff
@@ -175,6 +202,45 @@ func (r *homeChainPoller) setState(chainConfigs map[cciptypes.ChainSelector]Chai
 	s.fChain = createFChain(chainConfigs)
 }
 
+func (r *homeChainPoller) fetchAndSetRMNConfig(ctx context.Context) error {
+	var versionedConfigWithDigests []VersionedConfigWithDigest
+	err := r.homeChainReader.GetLatestValue(
+		ctx,
+		r.rmnHomeBoundContract.ReadIdentifier(consts.MethodNameGetVersionedConfigsWithDigests),
+		primitives.Unconfirmed,
+		map[string]interface{}{
+			"offset": 0,
+			"limit":  2, // TODO: fetch CONFIG_RING_BUFFER_SIZE
```

Comment:
I placed a comment on the contract PR but this API seems convoluted to me

Reply:
Based on our last meeting, the API is going to be changed. I'll update the code in an upcoming PR when changes are ready.